instrumentation/opentelemetry-instrumentation-logging/README.rst

```diff
@@ -13,6 +13,140 @@ Installation
 
     pip install opentelemetry-instrumentation-logging
 
+Log record enrichment behavior of logging instrumentation
+
+"""
+The OpenTelemetry ``logging`` integration automatically injects tracing context into log statements.
+It's structured within an `if` statement that checks  [`set_logging_format`] is `True`, indicating that logging configuration should be applied.
```

Comment:
I don't think implementation details like this is necessary to include in the README